docs/developer-guide/standardize-python-project-configuration.md

```diff
@@ -0,0 +1,99 @@
+---
+myst:
+  html_meta:
+    "description": "Standardize project configuration in Plone with meta"
+    "property=og:description": "Standardize project configuration in Plone with meta"
+    "property=og:title": "Standardize project configuration in Plone with meta"
+    "keywords": "Plone 6, standardize, project, configuration, development, meta"
+---
+
+# Standardize Python project configuration
+
+This part of the documentation describes how to standardize Python project configuration in Plone.
+It does not cover Volto or any other JavaScript-based project, which has its own ecosystem.
+
+Plone consists of hundreds of projects.
+To lessen the effort of configuring a new project in the Plone GitHub organization, and to keep these projects current with latest configuration practices, the Plone community agreed upon a trusted set of configuration items.
+The Plone community manages these configuration items using the [`meta`](https://github.com/plone/meta) project.
+
+You can follow these practices in your own projects, or suggest new or alternative configuration items through the `meta` repository, sharing them with the rest of the Plone community.
```

Comment:
I would call it `plone/meta` or `plone.meta` through, rather than just `meta`. There are too many Metas to not disambiguate!

I think we should mention some limitations. It's currently only for use with repositories that have a single Python package at the top level of the repository (so, it can't be used with a monorepo that includes both frontend and backend, like the ones produced by cookieplone). It also doesn't currently work very well for projects that want to support multiple versions of Plone in the same branch.

Reply:
Done in 1657f29d4293e4fd4c3420c94013894d6267c25d